BookReader/plugins/plugin.url.js

```diff
@@ -104,7 +105,17 @@ BookReader.prototype.updateLocationHash = function(skipAnalytics) {
               values.details=(!values.offsite  &&  window.top.location.pathname.match(/^\/details\//)   ? 1 : 0);
             } catch (e){} //avoids embed cross site exceptions -- but on (+) side, means it is and keeps marked offite!
 
+            // Send bookreader ping
             archive_analytics.send_ping(values, null, 'augment_for_ao_site');
+
+            // Also send tracking event ping
+            archive_analytics.send_ping({
+                kind: 'event',
+                ec: 'BookReader',
+                ea: 'UserChangedView',
+                el: location.pathname,
```

Comment:
Question: Do you want to be sending the path here? You could for example, send something like `this.pageIndex` (iirc).

Also, shouldn't it be `window.location.pathname` to be safe?

Reply:
Sure. Fixed.